src/controllers/v2/scheduled.js

```diff
@@ -61,6 +170,40 @@ const ScheduledController = {
     });
   },
 
+  /**
+   * Retrieve all registrations from the database that are expired.
+   *
+   * @returns {Sequelize.Model} All registrations that are expired.
+   */
+  async findAllExpiredNoRenewals(todaysDate) {
```

Comment:
If this is always going to be todays date, it would be better define it in the function rather than passing it in